prefs.js

```diff
@@ -17,17 +17,31 @@ function init() {
 function buildPrefsWidget() {
   let settings = Convenience.getSettings();
 
-  let widget = new Gtk.VBox();
-  widget.margin = 10;
-
-  addShortcut(widget, settings);
-  addImmediately(widget, settings);
-  addFontSize(widget, settings);
-  addMaxWidth(widget, settings);
-  addActivateByKey(widget, settings);
-
-  widget.show_all();
-  return widget;
+  let vwidget = new Gtk.VBox({margin: 10});
+
+  addShortcut(vwidget, settings);
+  addImmediately(vwidget, settings);
+
+  let hwidget = new Gtk.HBox();
+  
+  let fontSizeWidget = new Gtk.VBox({'margin-right': 10});
+  addFontSize(fontSizeWidget, settings);
+  fontSizeWidget.show_all();
```

Comment:
These extra `show_all`s are not needed, the single invocation on `vwidget` below recursively shows everything.